redfish-core/lib/account_service.hpp

```diff
@@ -1541,6 +1541,138 @@ inline void updateUserProperties(
     });
 }
 
+inline void uploadACF(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                      const std::vector<uint8_t>& decodedAcf)
+{
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec,
+                    sdbusplus::message::message& m,
+                    const std::tuple<std::vector<uint8_t>, bool, std::string>&
+                        messageFDbus) {
+        if (ec)
+        {
+            BMCWEB_LOG_ERROR("DBUS response error: {}", ec);
+            if (strcmp(m.get_error()->name, "xyz.openbmc_project.Certs.Error."
+                                            "InvalidCertificate") == 0)
+            {
+                redfish::messages::invalidUpload(
+                    asyncResp->res,
+                    "/redfish/v1/AccountService/Accounts/service",
+                    "Invalid Certificate");
+            }
+            else
+            {
+                messages::internalError(asyncResp->res);
```

Comment:
Please add `BMCWEB_LOG_ERROR() with ec.value()` prior to `internalError()`

Reply:
We have 
BMCWEB_LOG_ERROR("DBUS response error: {}", ec);  <---1571 , is not ok?

Reply:
You're right. It is already there before internalError.